nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -69,12 +69,18 @@ Use `configure.packages` instead.
 
 ## Other Notable Changes {#sec-release-22.11-notable-changes}
 
+<<<<<<< HEAD
```

Comment:
Fix it here and regenerate the docbook files.

Reply:
yes, on it